benchmark/torch/ppo/train.py

```diff
@@ -70,9 +70,15 @@ def main():
     else:
         model = AtariModel(obs_space, act_space)
 
-    ppo = PPO(model, config['clip_param'], config['value_loss_coef'],
-              config['entropy_coef'], config['initial_lr'], config['eps'],
-              config['max_grad_norm'], continuous_action=args.continuous_action)
+    ppo = PPO(
+        model,
+        config['clip_param'],
```

Comment:
Can we just set all these hyper-parameters as default values in the PPO algorithm?

Reply:
Change to only pass in parameters that are different in mujoco and atari environments.